### PR TITLE
List view: link to venue, show Gmap link | #39905

### PR DESCRIPTION
**Pull request by barryhughes**
Further tweaks following QA feedback to ensure single events within list view display as previously.

[#39905](https://central.tri.be/issues/39905)

### CODE REVIEW

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [single-event.php:58](https://github.com/moderntribe/the-events-calendar/blob/10eebd3d0bc1a5dee9a40275b5fe60447f97d35f/src/views/list/single-event.php#L58)

> via jenkins/codesniffer-phpcs

File: src/functions/template-tags/venue.php [hunk 1]
```diff
@@ -460,8 +460,8 @@ function tribe_get_venue_details( $post_id = null ) {
 
 		$venue_details = array();
 
-		if ( $venue_name = tribe_get_venue( $post_id ) ) {
-			$venue_details['name'] = $venue_name;
+		if ( $venue_link = tribe_get_venue_link( $post_id ) ) {
+			$venue_details[ 'linked_name' ] = $venue_link;
```
Comment (borkweb) [inline src/functions/template-tags/venue.php]: Array indexes that are strings shouldn't have the extra spacing inside the square brackets like indexes composed of variables/concatenation. Ref: [WordPress standards on spacing](https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#space-usage)

File: src/views/list/single-event.php [hunk 2]
```diff
@@ -16,7 +16,7 @@
 $venue_details = tribe_get_venue_details();
 
 // Venue
-$has_venue_address = ( ! empty( $venue_details['address'] ) ) ? ' location' : '';
+$has_venue_address = ( ! empty( $venue_details[ 'address' ] ) ) ? ' location' : '';
```
Comment (borkweb) [inline src/views/list/single-event.php]: Same spacing comment for the index as above.

**Comment by barryhughes** [comment]
Tidied!

**Comment by borkweb** [comment]
:+1:

**Pull request merged**